wikimedia/wikipedia-ios
# Add Watchlist onboarding modal

**Phabricator:** https://phabricator.wikimedia.org/T335609

### Notes
* This PR depends on https://github.com/wikimedia/wikipedia-ios-components/pull/20, make sure that PR is merged and the package commit is updated
* This PR adds the onboarding modal to the Watchlist feature

### Test Steps
1. Go to account, click on watchlists
2. The onboarding modal should display
3. Click continue to proceed to the watchlist feature
4. Click learn more to go to a special page (NOTE: I'm using a temporary URL here until I get the correct one)
5. Make sure the layout is correct according to the Figma file, and theming works
